src/json/brace-data-exchange.schema.json

```diff
@@ -1636,6 +1667,31 @@
           "title": "Environmental Problems",
           "description": "The property has problems that are environmental in nature (e.g. black mold.)"
         },
+        "has_new_principal_residence_over_100mi_away": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has New Principal Residence Over 100 Miles Away",
+          "description": "The borrower or co-borrower has established a new principal residence more than 100 miles from the borrower’s current, or former, principal residence. "
+        },
+        "has_new_dependents_property_doesnt_meet_needs": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has New Dependents property Doesn't Meet Needs",
+          "description": "Borrower to provide satisfactory evidence that there has been an increase in legal dependents and the principal residence fails to meet family needs. "
+        },
+        "is_borrower_vacating_residence": {
+          "$ref": "#/definitions/tribool",
+          "title": "Is Borrower Vacating Residence",
+          "description": "The borrower or co-borrower is vacating the principal residence with no intent to return. The principal residence will remain occupied by at least one existing borrower.  "
```

Comment:
This is _also_ super nitpicky, but could we get rid of the trailing whitespace in all of the descriptions?

Reply:
Done.